doc/ctc-meetings/2016-01-27.md

```diff
@@ -0,0 +1,295 @@
+# Node Foundation CTC Meeting 2016-01-27
+
+## Links
+
+* **Audio Recording**: https://soundcloud.com/node-foundation/ctc-meeting-2016-01-27
+* **GitHub Issue**: https://github.com/nodejs/node/issues/4901
+* **Minutes Google Doc**: <https://docs.google.com/document/d/145ND-9pGdAwZ1eoOMypX1_wtx5FCVoanWtLl8zzyxH8>
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1X3RbJhhLvFNojmQwxmsfUb2qPMyNPcVzcSS2aAoAon0>_
+
+## Present
+
+* James Snell (CTC)
+* Trevor Norris (CTC)
+* Colin Ihrig (CTC)
+* Brian White (CTC)
+* Alexis Campailla (CTC)
+* Bert Belder (CTC) x
+* Chris Dickinson (CTC)
+* Shigeki Ohtsu (CTC) x
+* Steven Loomis (observer) x
+* Mikeal Rogers (observer)
+* Fedor Indutny (CTC)
+* Jeremiah Senkpiel (CTC)
+* Rod Vagg (CTC)
+* Ben Noordhuis (CTC)
+* Domenic Denicola (observer) x
+* Nikita Skovoroda (observer)
+* Ali Sheikh (observer) x
+* Evan Lucas (observer)
+* Rich Trott (observer)
+* Michael Dawson (observer)
```

Comment:
I wasn't in this meeting but I'm taking it to mean not-present and we continued this _convention_ in the last meeting as well. Seems like that's not clear so do you have a better suggestion that's simple for note-takers? Deleting names has proven to be annoying for various reasons, although I guess I could remove them when I PR the minutes.